firefox_puppeteer/ui/toolbars.py

```diff
@@ -12,179 +12,187 @@
-    navigation bar as well as the location bar."""
-
+        navigation bar as well as the location bar."""
+    
     @property
     def back_button(self):
         """Provides access to the DOM element back button in the navbar.
-
-        :returns: Reference to the back button.
-        """
+            
+            :returns: Reference to the back button.
+            """
         return self.marionette.find_element(By.ID, 'back-button')
-
+    
     @property
     def forward_button(self):
         """Provides access to the DOM element forward button in the navbar.
-
-        :returns: Reference to the forward button.
-        """
+            
+            :returns: Reference to the forward button.
+            """
         return self.marionette.find_element(By.ID, 'forward-button')
-
+    
     @property
     def home_button(self):
         """Provides access to the DOM element home button in the navbar.
-
-        :returns: Reference to the home button element
-        """
+            
+            :returns: Reference to the home button element
+            """
         return self.marionette.find_element(By.ID, 'home-button')
-
+    
     @use_class_as_property('ui.toolbars.LocationBar')
     def locationbar(self):
         """Provides access to the DOM elements contained in the
-        locationbar.
-
-        See the :class:`LocationBar` reference.
-        """
-
+            locationbar.
+            
+            See the :class:`LocationBar` reference.
+            """
+    
     @property
     def menu_button(self):
         """Provides access to the DOM element menu button in the navbar.
-
-        :returns: Reference to the menu button element.
-        """
+            
+            :returns: Reference to the menu button element.
+            """
         return self.marionette.find_element(By.ID, 'PanelUI-menu-button')
 
 
 class LocationBar(BaseLib):
     """Provides access to and methods for the DOM elements contained in the
-    locationbar (the text area of the ui that typically displays the current url)."""
-
+        locationbar (the text area of the ui that typically displays the current url)."""
+    
     dtds = ["chrome://branding/locale/brand.dtd",
             "chrome://browser/locale/browser.dtd"]
-
-    def __init__(self, *args, **kwargs):
-        BaseLib.__init__(self, *args, **kwargs)
-        # TODO: A "utility" module that sets up the client directly would be
-        # useful here.
-        self.l10n = L10n(self.get_marionette)
-        self.keys = Keys(self.get_marionette)
-
-    @use_class_as_property('ui.toolbars.AutocompleteResults')
+        
+            def __init__(self, *args, **kwargs):
+                BaseLib.__init__(self, *args, **kwargs)
+                    # TODO: A "utility" module that sets up the client directly would be
+                    # useful here.
+                    self.l10n = L10n(self.get_marionette)
+                        self.keys = Keys(self.get_marionette)
```

Comment:
Same here. This is even code with broken indenation, which even Python should complain about. This definitely needs a fix.